pytest_rts/pytest/runner_plugin.py

```diff
@@ -0,0 +1,30 @@
+"""This module contains code for running a specific test set"""
+# pylint: disable=too-few-public-methods
+from typing import List
+
+from _pytest.main import Session
+from _pytest.nodes import Item
+
+from pytest_rts.pytest.fake_item import FakeItem
+from pytest_rts.utils.common import filter_pytest_items
+
+
+class RunnerPlugin:
+    """Plugin class for pytest"""
+
+    def __init__(self, existing_tests) -> None:
```

Comment:
Why the type for `existing_tests` is omitted? There are several more missing type hints in the PR.

Reply:
Maybe I'm a bit confused about the type hints in python. Are they to be added for eveything? For example, in this line my editor shows the type `set` for it even without the type hint here. My thinking was that they're not required for places where this happens.

Reply:
I would say that type hints supposed to be everywhere. Maybe @alexey-vyskubov could clarify it a bit more.